pyproject.toml

```diff
@@ -25,6 +25,7 @@ mailchimp3 = "^3.0"
 python-dotenv = "^0.10.1"
 requests = "^2.21"
 sentry-sdk = "^0.7.14"
+bandit = "~1.5.1"
```

Comment:
We only need this declared as a dev-dependency since it's not needed at runtime.